### ISSUE TITLE
Documentation for V2

### ISSUE THREAD

**Issue by b-strauss** [polyfillpolyfill/polyfill-service#1867]
The V2 documentation is gone. We still have many projects using V2. Are the V2 docs accessible somewhere?

**Comment by jonscottclark** [comment]
Agreed that the old docs should be made available. Only solution at the moment is to search for "site:polyfill.io v2" in Google and view cached results.

**Comment by ianwremmel** [comment]
Any update on when documentation will return?

**Comment by sleagon** [comment]
missing

**Comment by ianwremmel** [comment]
Where are the restored v2 docs hosted?
